InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -4020,7 +4020,7 @@ private void StartAnimation(SceneObjectPart part, string anim)
                         result = presence.AddAnimation(anim, m_host.UUID);
                     else
                         result = presence.AddAnimation(animID, m_host.UUID);
-                    if (result != String.Empty)
+                    if (!String.IsNullOrEmpty(result))
```

Comment:
No semantic change as none of the overloads of `ScenePresence::AddAnimation()` can return a null,